Makefile

```diff
@@ -112,10 +112,10 @@ demo: ## make new demo
 	@$(RUN_ADMIN) ems:admin:restore --configs-folder=./demo/configs/admin --configs --force
 	@$(RUN_ADMIN) ems:contenttype:activate --all --force --no-debug
 	@$(RUN_ADMIN) ems:environment:rebuild --all --no-debug
-	@$(RUN_ADMIN) ems:managed-alias:add-environment ma_preview preview
-	@$(RUN_ADMIN) ems:managed-alias:add-environment ma_preview default
-	@$(RUN_ADMIN) ems:managed-alias:add-environment ma_live live
-	@$(RUN_ADMIN) ems:managed-alias:add-environment ma_live default
+	@$(RUN_ADMIN) ems:managed-alias:add-environment ma-preview preview
+	@$(RUN_ADMIN) ems:managed-alias:add-environment ma-preview default
+	@$(RUN_ADMIN) ems:managed-alias:add-environment ma-live live
+	@$(RUN_ADMIN) ems:managed-alias:add-environment ma-live default
```

Comment:
This is wrong PR incoming

Reply:
Do you want that I rollback those lines?

Reply:
Yes, not needed anymore. https://github.com/ems-project/elasticms/pull/1173 fixes this issue.